packages/marshal/docs/marshal-vs-store-level.md

```diff
@@ -0,0 +1,95 @@
+| Classification | `keyStyleOf(p)`<br>`patternStyleOf(p)` | `passStyleOf(p)`<br>`getInterfaceOf(j)` | `typeof j`                    |
+| Type Testing   | `isKey(p)`         | `isOnlyData(p)`<br>`isStructure(p)` | `isPrimitive(j)`<br>`Array.isArray(j)`                |
+| Equivalence    | `sameKey(k,k)`     | `sameStructure(s,s)`                | `j === j`<br>`Object.is(j,j)`<br>`sameValueZero(j,j)` |
+| Ordering       | `compareKeys(k,k)` | `comparePassables(p,p)`             | `j <= j`<br>`[...js].sort(fullCompare(j,j))`          |
+
+Where the parameter
+   * `j` is for any JavaScript value.
+   * `p` is for any `Passable`, a subset of JavaScript values.
+   * `s` is for any `Structure`, a subset of passables.
+   * `k` is for any `Key`, a subset of structures.
+
+
+|            | `keyStyleOf`            | `passStyleOf`          | `typeof`                                                     |
+| ---------- | ----------------------- | ---------------------- | ------------------------------------------------------------ |
+| Primitives | js primitives + `null`  | js primitives + `null` | `undefined` `boolean` `bigint`<br>`number` `string` `symbol` |
+| Containers | `copyArray` `copyRecord`<br>`copySet` `copyMap` | `copyArray` `copyRecord`<br>`copyTagged` | `object`           |
+| Behavior   | `remotable`             | `remotable`            | `function`                                                   |
+| Others     | No keys<br>See patterns | `error` `promise`<br>`metaTagged` |                                                   |
+
+
+
+## The Marshal level organized by `passStyleOf`
+
+We organize according to the classification from<br>
+**`passStyleOf(p: Passable) => PassStyle`**<br>
+where the returned `PassStyle` is one of a fixed enumeration of strings, similar to the strings returned by JavaScript's `typeof`.
+
+`PassStyle`:
+  * **Primitive styles**: `"undefined"`, `"null"`, `"boolean"`, `"bigint"`, `"number"`, `"string"`, `"symbol"`
+  * **Data styles**: The primitive styles + `"copyArray"`, `"copyRecord"`, `"copyTagged"`
+  * **Structure styles**: The data styles + `"remotable"`
+  * **`PassStyle`**: The structure styles + `"error"`, `"promise"`, `"metaTagged"`
+
+The **primitive styles** identify the *passable primitives*. The passable primitives have the expected mapping down to the JavaScript primitives. However, the passable primitives do not include JavaScript's `-0` or anonymous symbols. Those JavaScript primitives that are included round-trip through the protocol.
+
+The **data styles** tag data-like containers. If a data-like container contains only data, then it is only data, passing the `isOnlyData` type test.
+  * A *copyArray* is a simple JavaScript array containing only passables.
+  * A *copyRecord* is a simple JavaScript record-like object, with only string-named own data properties containing only passables.
+  * A *copyTagged* is the extensibility point used by the Store level to implement *copySets* and *copyMaps*.
```

Comment:
`copyTagged` needs more explanation.  You provide various bits of precise definitional language but never say what it actually _is_ in a more informal sense that people can actually understand.  That renders the narrative much harder to follow, when I expect (but am not sure) that it's actually fairly simple and straightforward.

Reply:
Agreed. Being very precise can actually make it harder for readers to understand, when the precision requires new terms and concepts. Being a little more imprecise and using analogies to orient the reader might be helpful.